### ISSUE TITLE
Proceed to Checkout button overlap on Firefox mobile

### ISSUE THREAD

**Issue by peterfabian** [woocommerce/woocommerce-gutenberg-products-block#2315]
**Describe the bug**
Proceed to checkout button overlaps with the Storefront overlay on Firefox mobile

**To Reproduce**

1. Add item to cart
2. Go to cart
3. See overlap, both with and without shipping debug.

**Expected behavior**
Something nicer. :)

**Screenshots**

![Screenshot_20200427-150958_Firefox](https://user-images.githubusercontent.com/2207451/80376136-a9e5b980-8899-11ea-82c3-3fe82b0a85d6.jpg)
![Screenshot_20200427-150906_Firefox](https://user-images.githubusercontent.com/2207451/80376148-ace0aa00-8899-11ea-8a51-43de60eebb80.jpg)

Core version: 5.4
WooCommerce version: 4.0.1
WooCommerce Blocks version: 2.6.0-beta.1
Site language: en_US
Any other plugins installed:
```
Active plugins (8)
--
Companion Plugin |   | by Osk – 1.11
Facebook for WooCommerce |   | by Facebook – 1.11.0
Jetpack by WordPress.com |   | by Automattic – 8.4.2
Mailchimp for WooCommerce |   | by Mailchimp – 2.3.6
WooCommerce Stripe Gateway |   | by WooCommerce – 4.3.3
WooCommerce Blocks |   | by Automattic – 2.6.0-beta.1
WooCommerce - ShipStation Integration |   | by WooCommerce – 4.1.35
WooCommerce |   | by Automattic – 4.0.1
```
**Smartphone (please complete the following information):**

* Device: Galaxy S8+
* OS: Android 9
* Browser Firefox 68.7.0

**Comment by nerrad** [comment]
I _think_ this might be fixed in the latest Storefront RC1 candidate, is that correct @Aljullu or @haszari? I also recall seeing another issue that was open regarding this and closed in favor of fixing in storefront?

**Comment by Aljullu** [comment]
Yes, it was fixed in https://github.com/woocommerce/storefront/pull/1307. The fix is included in Storefront 2.5.6 RC1.
